src/lms-harmonizer/tests_integration_sql/mssql_connection.py

```diff
@@ -0,0 +1,55 @@
+# SPDX-License-Identifier: Apache-2.0
+# Licensed to the Ed-Fi Alliance under one or more agreements.
+# The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+# See the LICENSE and NOTICES files in the project root for more information.
+# from os import path
+
+from contextlib import contextmanager
+from typing import Any, Dict, Iterator, List
+import pyodbc
```

Comment:
Dropped down to pyodbc as sqlalchemy had issues due to connection pool management.